packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FGMGroundOverlayController.h

```diff
@@ -0,0 +1,57 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <GoogleMaps/GoogleMaps.h>
+
+#import "messages.g.h"
+
+/// Controller of a single ground overlay  on the map.
+@interface FGMGroundOverlayController : NSObject
+
+/// The ground overlay this controller handles.
+@property(strong, nonatomic) GMSGroundOverlay *_Nonnull groundOverlay;
+
+/// Whether ground overlay is created with bounds or position.
+@property(nonatomic, assign) BOOL isCreatedWithBounds;
```

Comment:
Properties should not be named "is*"; see https://google.github.io/styleguide/objcguide.html#objective-c-method-names

Reply:
Changed it like this:
```objc
@property(nonatomic, assign, getter=isCreatedWithBounds) BOOL createdWithBounds;
```